WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderActivityLauncher.java

```diff
@@ -68,7 +68,7 @@ public static void showReaderPostPagerForTag(Context context,
         Intent intent = new Intent(context, ReaderPostPagerActivity.class);
         intent.putExtra(ReaderConstants.ARG_POST_LIST_TYPE, postListType);
         intent.putExtra(ReaderConstants.ARG_TAG, tag);
-        intent.putExtra(ReaderConstants.ARG_TITLE, tag.getTagName());
+        intent.putExtra(ReaderConstants.ARG_TITLE, tag.getCapitalizedTagName());
```

Comment:
It shouldn't be necessary to capitalize the tag in the intent. If that's being done to esnure captilization in the header, that should be done [here](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java#L977) instead, ex:
```
String htmlTag = "<font color=" + color + ">" + getCurrentTag().getCapitalizedTagName() + "</font>";
```
Note that you'll want to check `hasCurrentTag()` first to avoid a possible NPE.